src/osx/cocoa/listbox.mm

```diff
@@ -375,7 +381,7 @@ wxAlignment WXUNUSED(just), int defaultWidth)
         [col1 setMinWidth:10];
         // temporary hack, because I cannot get the automatic column resizing
         // to work properly
-        [col1 setWidth:1000];
+        // [col1 setWidth:1000];
```

Comment:
Could we get rid of this now?

Reply:
unfortunately this still is not properly resolved for the case when a horizontal scrollbar is wanted, then the column's size is only initially calculated correctly, if it later gets longer strings, the content will be eclipsed with '..' instead of increasing column size

https://trac.wxwidgets.org/ticket/18860